ote/src/cljs/ote/app/routes.cljs

```diff
@@ -130,7 +144,9 @@
                      orig-app app
                      app (merge app navigation-data)]
 
-                 (if (requires-authentication? app)
+                 (if (or (requires-authentication? app)
```

Comment:
Should functions return true if user passes a check and fail if not?
Should functions be renamed based on the check they make?

```suggestion
                 (if (or (not (page-user-authenticated? app))
                         (not (page-user-transit-authority-authorized? app))
                         (not (page-user-admin-authorized app))
```

Reply:
Functions returns boolean value true/false. It is indicated using ? in the function name. I think that functions are named well enough and for this time I'm not changing their names.